lte/gateway/c/core/oai/tasks/nas/esm/sap/esm_recv.c

```diff
@@ -646,8 +650,8 @@ status_code_e erab_setup_rsp_tmr_exp_handler(
   ue_mm_context_t* ue_mm_context = mme_app_get_ue_context_for_timer(
       ue_id, "EPS BEARER DEACTIVATE T3495 Timer");
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(
-        LOG_MME_APP,
+    OAILOG_ERROR_UE(
```

Comment:
ue_mm_context does not exist in this case, let's not change this

Reply:
Resolved